go/flags/endtoend/vtgr.txt

```diff
@@ -29,7 +29,6 @@ Usage of vtgr:
       --logtostderr                                log to standard error instead of files
       --mysql_server_flush_delay duration          Delay after which buffered response will be flushed to the client. (default 100ms)
       --mysql_server_version string                MySQL server version to advertise.
-      --pid_file string                            If set, the process will write its pid to the named file, and delete it on graceful shutdown.
```

Comment:
does `vtgr` not use `servenv`?

Reply:
it doesn't listen for any RPCs https://github.com/vitessio/vitess/blob/main/go/cmd/vtgr/main.go#L25-L41